modules/claims_api/app/services/claims_api/power_of_attorney_request_service/terminate_existing_requests.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'bgs_service/veteran_representative_service'
+require 'bgs_service/manage_representative_service'
+
+module ClaimsApi
+  module PowerOfAttorneyRequestService
+    class TerminateExistingRequests
+      DEFAULT_FIRST_NAME = 'vets-api'
+      DEFAULT_LAST_NAME = 'vets-api'
+      FORM_TYPE_CODE = '21-22'
+
+      def initialize(veteran_participant_id)
+        @veteran_participant_id = veteran_participant_id
+      end
+
+      def call
+        mapped_requests = []
+        requests = get_non_obsolete_requests
+        requests.each do |request|
+          mapped_requests << { proc_id: request['procId'],
+                               representative: { first_name: DEFAULT_FIRST_NAME,
+                                                 last_name: DEFAULT_LAST_NAME } }
+        end
+
+        mapped_requests.each do |request|
+          set_to_obsolete(request)
+        end
```

Comment:
lots of wsdl calls are going to happen here for Veterans with many open requests